src/test/ui/proc-macro/issue-91800.stderr

```diff
@@ -0,0 +1,56 @@
+error: macros that expand to items must be delimited with braces or followed by a semicolon
+  --> $DIR/issue-91800.rs:6:10
+   |
+LL | #[derive(MyTrait)]
+   |          ^^^^^^^
+   |
+   = note: this error originates in the derive macro `MyTrait` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: proc-macro derive produced unparseable tokens
+  --> $DIR/issue-91800.rs:6:10
+   |
+LL | #[derive(MyTrait)]
+   |          ^^^^^^^
+
+error: 
```

Comment:
What is going on here! 😮

Reply:
I can use a custom error message if necessary, it will be more clear.